script/plugin_webpack_directories.rb

```diff
@@ -39,8 +39,10 @@ def entry_name(plugin_name, entry)
 specs.each do |dep|
   # skip other rails engines that are not plugins
   # TODO: Consider using the plugin registration api?
-  next unless dep.name =~ plugin_name_regexp
-  next if dep.name =~ /.*[_-]core$/
+  unless dep.respond_to?("metadata") && dep.metadata.has_key?("is_foreman_plugin") && dep.metadata["is_foreman_plugin"] == "true"
+    next unless dep.name =~ plugin_name_regexp
+    next if dep.name =~ /.*[_-]core$/
```

Comment:
I wonder if we can use the plugin in webpack if it contains the `/webpack` dir, without using the regex for plugins that starts with `katello`/`foreman` or ends with `core`,
if it has the `/webpack` dir I think we can assume the plugin owners meant it to be used by webpack, what do you think?

Reply:
This is "old" code I just put in the metadata contains "is_foreman_plugin" check. I can change it to whatever you / the community like.

Reply:
thanks, I forgot that each plugin is processed as a gem here so we don't want an accidental gem that contains a random webpack dir, your solution with `is_foreman_plugin` looks like the best approach, ACK from me 👍🏼

Reply:
I certainly agree and IMHO we should deprecate the old name based method, which is why I suggested an RFC. That gives a lot more visibility.

Reply:
@ekohl you mean a RFC on community forum on discourse?

Reply:
Yes. This is IMHO a good change and it deserves more visibility. If no plugin author knows about this, it would be a waste.

Reply:
Done: https://community.theforeman.org/t/collect-plugins-which-should-be-processed-by-webpack/25567/2